app/common/src/main/AndroidManifest.xml

```diff
@@ -143,6 +143,11 @@
                 <data android:scheme="mailto" />
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERE" />
```

Comment:
The action name is missing a "D" at the end.

```suggestion
                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
```

Reply:
That was a dumb mistake. Fixed it!